.changeset/pre.json

```
@@ -0,0 +1,22 @@
+{
+  "mode": "pre",
+  "tag": "premajor",
```

We'll want to tag it with this

```suggestion
  "tag": "alpha-v5",
```